sycl/include/sycl/ext/intel/experimental/esimd/detail/math_intrin.hpp

```diff
@@ -449,7 +449,10 @@ __esimd_dpas_inner(const __ESIMD_DNS::vector_type_t<T0, SZ> *src0,
                                 __ESIMD_EMU_DNS::is_inttype<RT>::value>::set();
 
   constexpr __ESIMD_NS::uint ops_per_chan =
-      src1_precision == __ESIMD_ENS::argument_type::BF16 ||
+      src1_precision == __ESIMD_ENS::argument_type::TF32 ||
```

Comment:
Nit: seems this logic overlaps with __esimd_dpas_bits_precision. As a variant:
```
constexpr auto nbits1 = __esimd_dpas_bits_precision<src1_precision>();
constexpr auto nbits2 = __esimd_dpas_bits_precision<src2_precision>();
constexpr auto nbits = max(nbits1, nbits2);
constexpr __ESIMD_NS::uint ops_per_chan = std::min(32 / nbits, 8);
```

Reply:
Thank you. Fixed in c3b19813bd98fbefae6b1def219d2a9ee045978f